charts/testmachinery/charts/argo/values.yaml

```diff
@@ -21,16 +21,21 @@ argo:
   name: workflow-controller
   executor:
     containerRuntimeExecutor: docker
-  resources: {}
+  resources: { }
   # requests:
   #   cpu: 200m
   #   memory: 256Mi
   # limits:
   #   cpu: 1
   #   memory: 1Gi
 
+
   logging:
-    links: []
+    # default links to a grafana instance will be injected
+    grafana:
+      host: "grafana.example.com"
```

Comment:
not sure if we should default the values to something that does not exist.
I would rather comment it and only set if a host really available.

Reply:
Tricky question - so far those "default" logging links have been derived automatically by the dependency ensurer: https://github.com/gardener/test-infra/blob/master/pkg/testmachinery/controller/dependencies/argo.go#L99 
However I would agree, that we should make this optional. I've changed the chart structure slightly to accommodate it.
Unfortunately I cannot implement a check similar to the ensurer since the `lookup` function returns nil when we use `helm template` to render the chart for `kubectl apply`.